migaloo/assetlist.json

```diff
@@ -143,6 +143,62 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/rac.svg",
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/rac.png"
       }
+    },
+    {
+      "description": "The native stablecoin of Kava",
+      "denom_units": [
+        {
+          "denom": "usdx",
+          "exponent": 0
+        },
+        {
+          "denom": "USDX",
+          "exponent": 6
+        }
+      ],
+      "base": "usdx",
+      "name": "USDX",
+      "display": "USDX",
+      "symbol": "USDX",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kava/images/usdx.svg",
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kava/images/usdx.png"
+      },
+      "coingecko_id": "usdx"
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "ibc/80E8F826480B995AE28C1EE86106C1BE2034FF1966579D29951CF61885458040",
+          "exponent": 0,
+          "aliases": [
+            "uusdc"
+          ]
+        },
+        {
+          "denom": "usdc",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/80E8F826480B995AE28C1EE86106C1BE2034FF1966579D29951CF61885458040",
+      "name": "USD Coin",
+      "display": "usdc",
+      "symbol": "USDC",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "axelar",
+            "base_denom": "uusdc",
+            "channel_id": "channel-112"
```

Comment:
I couldn't find any axelar-migaloo ibc channel registered under the /IBC folder.